data-processing/tests/test_extract_definitions.py

```diff
@@ -244,6 +426,14 @@ def test_extract_abbreviation_acronym(model):
     assert abbreviations[0].definition_text == "Convolutional Neural Network"
 
 
+@pytest.mark.slow
+def test_model_extract_abbreviation_shortened_word(model: DefinitionDetectionModel):
+    prediction_type = 'ABBR-EXP'
+    features = model.featurize("We propose Conductive Networks (CondNets)")
+    intents, slots, _ = model.predict_batch([features], prediction_type)
+    assert intents[0]
+    assert slots[0] == ["O", "O", "DEF", "DEF", "O", "TERM", "O"]
+
 @pytest.mark.slow
 def test_extract_abbreviation_shortened_word(model):
```

Comment:
this file is starting to get a bit long; can you take a quick look and see if any of the tests are redundant, and if so, remove them?

Reply:
Will do!